pkg/artifactory/resource/repository/remote/remote.go

```diff
@@ -11,40 +11,38 @@ import (
-	FailedRetrievalCachePeriodSecs    int                                `json:"failedRetrievalCachePeriodSecs,omitempty"`
+	Key                               string                             `json:"key,omitempty"`
+	ProjectKey                        string                             `json:"projectKey"`
+	ProjectEnvironments               []string                           `json:"environments"`
+	Rclass                            string                             `json:"rclass"`
+	PackageType                       string                             `json:"packageType,omitempty"`
+	Url                               string                             `json:"url"`
+	Username                          string                             `json:"username"`
+	Password                          string                             `json:"password"`
+	Proxy                             string                             `json:"proxy"`
+	Description                       string                             `json:"description"`
+	Notes                             string                             `json:"notes"`
+	IncludesPattern                   string                             `json:"includesPattern"`
+	ExcludesPattern                   string                             `json:"excludesPattern"`
+	RepoLayoutRef                     string                             `json:"repoLayoutRef"`
+	RemoteRepoLayoutRef               string                             `json:"remoteRepoLayoutRef"`
+	HardFail                          *bool                              `json:"hardFail,omitempty"`
+	Offline                           *bool                              `json:"offline,omitempty"`
+	BlackedOut                        *bool                              `json:"blackedOut,omitempty"`
+	XrayIndex                         bool                               `json:"xrayIndex"`
+	PropagateQueryParams              bool                               `json:"propagateQueryParams"`
+	QueryParams                       string                             `json:"queryParams,omitempty"`
+	PriorityResolution                bool                               `json:"priorityResolution"`
+	StoreArtifactsLocally             *bool                              `json:"storeArtifactsLocally,omitempty"`
+	SocketTimeoutMillis               int                                `json:"socketTimeoutMillis"`
+	LocalAddress                      string                             `json:"localAddress"`
+	RetrievalCachePeriodSecs          int                                `hcl:"retrieval_cache_period_seconds" json:"retrievalCachePeriodSecs"`
 	MissedRetrievalCachePeriodSecs    int                                `hcl:"missed_cache_period_seconds" json:"missedRetrievalCachePeriodSecs"`
-	UnusedArtifactsCleanupEnabled     *bool                              `hcl:"unused_artifacts_cleanup_period_enabled" json:"unusedArtifactsCleanupEnabled,omitempty"`
```

Comment:
This one is removed completely. Is this correct?

Reply:
Yes